apple-pie/index.html

```diff
@@ -4,36 +4,50 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Apple Pie Recipe</title>
-    <!-- Remember to link your styles -->
+    <link rel="stylesheet" href="styles/style.css">;
```

Comment:
Be careful, that semicolon at the end of the line is showing in your app. We don't need semicolons in html files